website/src/css/customTheme.css

```diff
@@ -501,3 +502,40 @@ a:hover {
 .help-page .card .buttons {
   display: flex;
 }
+
+/*newsletter section*/
+
+.center-elem{
+  display: flex;
+  justify-content: center;
+}
+
+.news-logo{
+  margin-bottom: 20px;
+}
+
+.news-button{
+  border: #111827;
+  padding: 5px 30px;
+  border-style: solid;
+  border-radius: 9px;
+  background: #111827;
+  color: white;
+  margin: 4px;
+  text-decoration: none;
+  transition: all 0.3s;
+}
+.news-button:hover{
+  background: black;
+  color: white;
+  text-decoration: none;
+}
+
+.news-logo-svg{
+  height: 42px;
+}
+
+.news-text{
+  color: var(--color-primary);
+  text-align: center;
+}
```

Comment:
Hi, please add a blank line here, others LGTM.

Reply:
An extra line at the end right, line 542.

Reply:
![image](https://user-images.githubusercontent.com/2106987/109946179-ec2a2380-7d12-11eb-8d7e-81fe69b8f1a1.png)